src/components/request/Request.tsx

```diff
@@ -17,9 +18,14 @@ const Request = () => {
   const { rows, headerCells } = useRequestTableData(searchParams);
 
   return (
-    <div>
-      <Table title={"Input Data"} headerCells={headerCells} rows={rows} />
-    </div>
+    <Wrapper>
+      <RequestHero />
+      <TableSection>
+        <TableContentWrapper>
+          <Table title={"Input Data"} headerCells={headerCells} rows={rows} />
```

Comment:
i dont think we need chain id in the table

Reply:
It was in the data you sent me :) we can hide it though.

Reply:
yea it will be needed, but just not in the table afaik. you will need it to display what chain the request is on